datafusion/physical-plan/src/joins/hash_join.rs

```diff
@@ -1003,13 +1028,14 @@ impl HashJoinStream {
         cx: &mut std::task::Context<'_>,
     ) -> Poll<Option<Result<RecordBatch>>> {
         let build_timer = self.join_metrics.build_time.timer();
+        // build hash table from left (build) side, if not yet done
```

Comment:
that looks great @alamb 
Thanks for leaving comments where is build and where is a probe.